modules/packages/LinearAlgebra.chpl

```diff
@@ -2490,6 +2490,151 @@ proc cosm(A: []) throws {
   }
 }
 
+proc sum(A: [?D], param axis=0) where !isCSDom(D) {
```

Comment:
minor nit -- I might rename this `absSum` (or something like that) as opposed to `sum`. Especially if this is user-facing.

Reply:
A few other design questions -
1. Should this throw if `axis!=0,1`?
2. Should this be more general and handle summing over any subset of indices?

Reply:
Resolved 1, working on 2.